.github/workflows/build.yml

```
@@ -18,12 +18,22 @@ jobs:
         with:
           fetch-depth: 0 # Needed for SonarCloud
 
-      - name: Check PR size
+      - name: Get PR labels
         if: github.event_name == 'pull_request'
+        id: pr-labels
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:26: Double quote to prevent globbing and word splitting

(shellcheck)